src/spec/context.cr

```diff
@@ -166,6 +166,19 @@ module Spec
     def matches?(pattern, line, locations)
       false
     end
+
+    @@spec_nesting = false
```

Comment:
If we were to ever get to parallel test execution, would a class variable cause a problem?

Reply:
Yes

Reply:
Oh, actually, I was replying generically that class variables are to be used carefully in a parallel context. Not sure about this case, though